.gitignore

```diff
@@ -9,3 +9,6 @@
 **/uploads/*.txt
 *.pyc
 *.env
+package-lock.json
```

Comment:
@schwaaweb : removing this as package-lock.json is supposed to be checked in (though I m wondering why it didn't show up for others)